etc/profile-m-z/nodejs-common.profile

```diff
@@ -1,5 +1,6 @@
 # Firejail profile for Node.js
-# Description: Common profile for npm/yarn
+# Description: Asynchronous event-driven JavaScript runtime
+# See comments for enabling debugging with certain IDEs
```

Comment:
I don't think this is necessary, we have comments like (un)comment for edge-case bla in a lot profiles.